CHANGELOG.md

```diff
@@ -49,6 +49,9 @@ BUG FIXES:
 * cloud: fixed a bug that would prevent nested symlinks from being dereferenced into the config sent to Cloud ([#686](https://github.com/opentofu/opentofu/issues/686)) 
 * cloud: state snapshots could not be disabled when header x-terraform-snapshot-interval is absent ([#687](https://github.com/opentofu/opentofu/issues/687))
 * Fixed crash during tofu destroy inside module with variable validation ([#817](https://github.com/opentofu/opentofu/issues/817))
+* Fix regression in ignoring GPG signing key expiration by default ([822](https://github.com/opentofu/opentofu/issues/822))
```

Comment:
Let's try and keep this past-tense like the rest. Maybe something like this could work?
```suggestion
* Fixed regression where `tofu` was ignoring GPG signing key expiration by default ([822](https://github.com/opentofu/opentofu/issues/822))
```